README.md

```diff
@@ -241,16 +241,38 @@ This service is responsible for running a short-circuit analysis and storing res
 
 This service is responsible for managing a file system like the hierarchy of directories where other kind of data managed by other micro-services (studies, contingency lists, filters called directory elements) can be referenced using their unique IDs. When added to a directory an element is associated to a name, an owner and a description. Access rights are only managed through directories and in the current version they are only implemented by a "private" attribute. A private directory is accessible with all its contents (some elements, or some sub-directories) to its owner only. A non private directory is implicitly a public directory and access is granted to all users.
 
+### Time-series server
+
+- Kind: Web service with a REST API
+- Source repository: https://github.com/gridsuite/timeseries-server
+- Storage: PostgreSQL
+- Connected to message broker: no
+- Other services dependencies:
+- Use PowSyBl libraries: yes
+
+This service is responsible for storing time-series data. It is used to persist the output data of dynamic simulations which are displayed later in GridStudy.
+
 ### Dynamic simulation server
 
 - Kind: Web service with a REST API
 - Source repository: https://github.com/gridsuite/dynamic-simulation-server
 - Storage: PostgreSQL
 - Connected to message broker: producer and consumer
-- Other services dependencies: network store server
+- Other services dependencies: network store server, report server, dynamic mapping server, time-series-server
+- Use PowSyBl libraries: yes
+
+This service is responsible for running dynamic simulations. It is based on [PowSyBl dynamic simulation API](https://github.com/powsybl/powsybl-core/tree/main/dynamic-simulation/dynamic-simulation-api) and [Dynawo implementation](https://github.com/powsybl/powsybl-dynawo). This is still an ongoing work and it still doesn't work completely with the current version of the code. This is not yet completely integrated into GridStudy.
```

Comment:
```suggestion
This service is responsible for running dynamic simulations. It is based on [PowSyBl dynamic simulation API](https://github.com/powsybl/powsybl-core/tree/main/dynamic-simulation/dynamic-simulation-api) and [Dynawo implementation](https://github.com/powsybl/powsybl-dynawo). 
```

Reply:
This documentation is not updated enough to keep some "it is ongoing work...". Let's make it simple so we don't need to keep it up to date later on this point. 
And for the opensource point of view, dynamic simulation is integrated